.github/workflows/slack-notification.yml

```diff
@@ -42,14 +42,22 @@ jobs:
         run: source .github/scripts/trigger-and-wait.sh
         env:
           INPUT_WORKFLOW_FILE_NAME: oclif-release.yml
```

Comment:
add the changes to slack notification as well. thanks @onuzbee for catching it. can view the test run [here](https://github.com/kridai/twilio-cli/runs/4533459575?check_suite_focus=true)

Reply:
Any reason why macos release is failing? Is this expected?

Reply:
+1

Reply:
yes, its expected, its failing on notariztion due to keytar dependency in twilio-cli-core. see the run [here](https://github.com/kridai/twilio-cli/runs/4533465381?check_suite_focus=true). In the next release tomorrow we wont be adding optional dependency in twilio-cli-core npm release (make install will not include optional dependencies), so that should fix this failure.